gpii/node_modules/deviceReporter/src/DeviceReporter.js

```diff
@@ -88,6 +88,12 @@ gpii.deviceReporter["static"].get = function (requestProxy, deviceReporter) {
     var promise = deviceReporter.installedSolutionsDataSource.get();
     promise.then(function (solutions) {
         deviceReporter.fireResponse(requestProxy, solutions);
+    }, function (err) {
+        requestProxy.events.onError.fire({
```

Comment:
Please use the request rather than the requestProxy - as you discovered, the proxy fires in peculiar ways which don't mirror those in the real request. Also, it will be abolished soon